src/pages/Topic/Topic.tsx

```diff
@@ -179,21 +158,30 @@ export const Topic = ({ useTopic = _useTopic }: TopicProps): JSX.Element => {
           fitViewOptions={{
             padding: 5,
             minZoom: 0.75,
-            nodes: [{ id: initialNodes[0].id }]
+            nodes: [{ id: initialNodes[0]?.id }]
           }}>
           <ResponsiveMiniMap />
           <Background gap={16} />
-          <Panel position="top-right">
-            <LabeledSwitch
-              labelLeft={t('pages.topic.grouped')}
-              labelRight={t('pages.topic.single')}
-              isGrouped={isGrouped}
-              setIsGrouped={setIsGrouped}
-            />
-          </Panel>
-          <Controls showInteractive={false} position="top-right" style={{ marginTop: 50 }} />
+          {
+            <Panel position="top-right">
```

Comment:
könnte man als react flow navigationbutton machen und da aktivieren und deaktivieren

Reply:
bugged rum in react flow navigationbutton, sah scheiße aus "julian"